aries_cloudagent/protocols/endorse_transaction/v1_0/routes.py

```diff
@@ -163,6 +187,9 @@ async def transaction_create_request(request: web.BaseRequest):
     connection_id = request.query.get("conn_id")
```

Comment:
We don't need to ask for the Connection ID again, since we get it as part of the schema or cred def request (it should be validated at that point).

Reply:
Connection_id is removed, while creating request, and it is being assigned to transaction_record during creation of record.